cfme/tests/intelligence/reports/test_reports.py

```diff
@@ -116,9 +117,7 @@ def create_custom_tag(appliance):
     tag = appliance.rest_api.collections.tags.action.create(
         {
             "name": "{cat_name}_entry".format(cat_name=category_name),
-            "description": "{cat_name}_entry_description".format(
-                cat_name=category_name
-            ),
+            "description": "{cat_name}_entry_description".format(cat_name=category_name),
```

Comment:
While you are at it ,let's use f string

Reply:
Done! And modified all the other strings as well.